ocs_ci/ocs/platform_nodes.py

```diff
@@ -42,7 +42,6 @@ class NodesBase(object):
 
     """
     def __init__(self):
-        self.cluster_nodes = get_node_objs()
```

Comment:
We're sure this isn't needed before `restart_nodes_teardown` is called?

Reply:
yeah I have tested it out, its just moved down the function where its needed.